spec/lib/vanagon/component/source/git_spec.rb

```diff
@@ -25,6 +25,12 @@
         .to eq('facter')
     end
 
+    it "ref defaults to HEAD when no ref is passed in" do
+      git_source = klass.new(url, workdir: workdir)
+      expect(git_source.ref)
+        .to eq('HEAD')
+    end
```

Comment:
Didn't I leave a comment about moving this into its own `#describe` block the other night? Has there been a rebase or were the comments lost?